addons/website_forum/views/website_forum.xml

```diff
@@ -808,6 +813,38 @@
     </div>
 </template>
 
+<!-- Template for user tooltip  -->
+<template id="forum_user_tooltip">
```

Comment:
Clean the structure, you have div inside div, holding div holding h ... try to have the minimal number of containers.